pages/api/account/manage/event/[[...data]].js

```diff
@@ -114,14 +119,23 @@ export async function updateEventApi(username, id, updateEvent) {
     log.error(e, error);
     return { error };
   }
+  
+  // Add to Changelog
+  logChange(await getServerSession(context.req, context.res, authOptions), {
+    model: "Event", 
+    changesBefore: beforeUpdate, 
```

Comment:
`beforeUpdate` is not used anywhere else, so I suggest it can be moved from line 65 to here similar to line 127 with

```suggestion
    changesBefore: await getEventApi(username, id);
```